roboteam_ai/src/control/positionControllers/PosController.h

```diff
@@ -8,22 +8,25 @@
 #include "PosVelAngle.h"
 #include <roboteam_ai/src/utilities/Constants.h>
 #include <roboteam_ai/src/control/pid.h>
-#include <roboteam_ai/src/world/WorldData.h>
 
 namespace rtt {
 namespace ai {
+
+namespace world {
```

Comment:
Why do you need to forward declare this??? Just include World?

Reply:
no can't.

circular dependencies

Reply:
um no

Reply:
If this creates a circular dependency you are doing something gravely wrong

Reply:
disagree.